node/services/config.go

```diff
@@ -21,12 +22,22 @@ type Config struct {
 	// Note: The trusted does *not* imply Headers are not verified, but trusted as reliable to fetch headers
 	// at any moment.
 	TrustedPeers []string
+	// PeersLimit defines how many peers will be added during discovery.
+	PeersLimit uint
+	// DiscoveryInterval is an interval between discovery sessions.
+	DiscoveryInterval time.Duration
+	// AdvertiseInterval is a interval between advertising sessions.
+	// NOTE: only full and bridge can advertise themselves.
+	AdvertiseInterval time.Duration
```

Comment:
Might be worth mentioning that this all relates to Full Node discovery within share pkg, not some general discovery.

Reply:
done